### PR TITLE
[TASK] Remove unneeded 'horizbuttons' reference

### PR DESCRIPTION
**Pull request by garvinhicking**
This seems like a leftover. I'm reviewing usage of "horizbuttons" in core docs, and this shouldn't need to exist here.

Releases: main, 12.4, 11.5

### CODE REVIEW

**Review by brotkrueml — APPROVED** [review]
Okay for me, but I think the whole page should be removed, as it serves only as a placeholder for some content which was moved some years ago.

**Comment by garvinhicking** [comment]
Hhhm. I wonder, shouldn't we re-move the whole section https://docs.typo3.org/m/typo3/reference-coreapi/main/en-us/ApiOverview/DataFormats/T3datastructure/Index.html one level up? There's only this sublevel then.

Another way is to remove the sentence "This content has moved to" and just leave the link list as a start page here?

I'm a bit confused what to do here. :D

**Comment by brotkrueml** [comment]
Have to look at it, haven't known this page since today ;-)

**Comment by brotkrueml** [comment]
Actually, I would move the subpages into the FlexForms chapter where it belongs IMHO. Then remove the "Data formats" page.

**Comment by garvinhicking** [comment]
@brotkrueml Good idea. I can do that and add redirects. Backport down to 11.5?

**Comment by brotkrueml** [comment]
> Backport down to 11.5?

Personally, yup, I would try that. If it is to difficult for 11.5 you can abandon then the backport.

**Comment by garvinhicking** [comment]
@brotkrueml I've nove moved the sections and placed links, can you check if that's ok?

I would then redirect the following links (main, 12.4, 11.5):

https://docs.typo3.org/m/typo3/reference-coreapi/main/en-us/ApiOverview/DataFormats/Index.html
-> https://docs.typo3.org/m/typo3/reference-coreapi/main/en-us/ApiOverview/FlexForms/T3datastructure/Index.html

https://docs.typo3.org/m/typo3/reference-coreapi/main/en-us/ApiOverview/DataFormats/T3datastructure/Index.html
-> https://docs.typo3.org/m/typo3/reference-coreapi/main/en-us/ApiOverview/FlexForms/T3datastructure/Index.html

https://docs.typo3.org/m/typo3/reference-coreapi/main/en-us/ApiOverview/DataFormats/T3datastructure/Elements/Index.html
-> https://docs.typo3.org/m/typo3/reference-coreapi/main/en-us/ApiOverview/FlexForms/T3datastructure/Elements/Index.html

https://docs.typo3.org/m/typo3/reference-coreapi/main/en-us/ApiOverview/DataFormats/T3datastructure/SheetReferences/Index.html
-> https://docs.typo3.org/m/typo3/reference-coreapi/main/en-us/ApiOverview/FlexForms/T3datastructure/SheetReferences/Index.html

https://docs.typo3.org/m/typo3/reference-coreapi/main/en-us/ApiOverview/DataFormats/T3datastructure/Parsing/Index.html
-> https://docs.typo3.org/m/typo3/reference-coreapi/main/en-us/ApiOverview/FlexForms/T3datastructure/Parsing/Index.html

**Comment by garvinhicking** [comment]
@brotkrueml Should we try this merge?

**Review by linawolf — DISMISSED** [review]

File: Documentation/ApiOverview/FlexForms/Index.rst [hunk 1]
```diff
@@ -9,6 +9,9 @@ FlexForms
 FlexForms can be used to store data within an XML structure inside a single DB
 column.
 
+More information on this Data Structure is available in the section
```
Comment (brotkrueml) [inline Documentation/ApiOverview/FlexForms/Index.rst:12]:
```suggestion
More information on this data structure is available in the section
```

**Review by brotkrueml — APPROVED** [review]

**Comment by garvinhicking** [comment]
Created intercept redirects for all 5 Links for 3 versions = 15 Redirects

**Pull request merged**